src/Directory.Build.props

```diff
@@ -13,18 +13,25 @@
     <FileVersion Condition="'$(VersionSuffix)' != '' AND '$(BuildNumber)' != ''">$(VersionPrefix).$(BuildNumber)</FileVersion>
```

Comment:
https://github.com/nhibernate/nhibernate-core/blob/master/build-common/NHibernate.props#L2

What about NHibernate.dev.props? Should it also be renamed to Directory.Build.dev.props?

Reply:
I don't know.

Reply:
Ok, renamed

Reply:
Maybe we should keep NHibernate.props in place. We can just import it here instead?

Reply:
Why? I don't like it. IMO, better to stick to the standard way of doing things.

Reply:
Because I don't like looking at different places in different branches. In particular that I need to modify different dev.props files.

We still do it standard way  by importing our Nhibernate.props inside Directory.Build.props. And keep things the same across all branches.

Reply:
Maybe I just revert `.dev.props` change then?

Reply:
What do you propose to have in Nhibernate.props?

Reply:
> What do you propose to have in Nhibernate.props?


I propose don't change Nhibernate.props. And content of `Directory.Build.props` should look something like:
```xml
<Project>
  <Import Project="../build-common/NHibernate.props"/>
</Project>
```

Reply:
![image](https://user-images.githubusercontent.com/144791/144995053-d6eb65fe-6c24-4940-9105-b71449a0b80d.png)

Reply:
`NHibernate.props` restored.